### PR TITLE
Editing list of available versions

### PR DESCRIPTION
**Pull request by NivekT**
Adding 0.3.0 to this page - https://pytorch.org/data/versions.html

The default page that users reach when they visit pytorch.org/data will still be the documentation for the`main` branch. I recommend keeping it that way unless you have other thoughts.

### CODE REVIEW

**Review by ejguan — COMMENTED** [review]
And, can you please update https://github.com/pytorch/data/blob/gh-pages/index.html to point to 0.3.0?

**Comment by NivekT** [comment]
> And, can you please update https://github.com/pytorch/data/blob/gh-pages/index.html to point to 0.3.0?

That's the part I am unsure about, it depends how much our API/doc will change in the very near future, but I can do that.

**Comment by NivekT** [comment]
It is done. The landing page will now redirect to `beta` which redirects to `0.3.0`.

File: beta [hunk 1]
```diff
@@ -0,0 +1 @@
+0.3.0
```
Comment (ejguan) [inline beta:1]: Symbolic link

File: index.html [hunk 2]
```diff
@@ -4,9 +4,8 @@
         <title>TorchData</title>
     </head>
     <body>
-        <p>The documentation is <a href="/data/main/index.html">here</a></p>
+        <p>The documentation is <a href="/data/beta/index.html">here</a></p>
```
Comment (ejguan) [inline index.html:7]: If you want to do so, you need to change `beta` as a symbolic link to `0.3.0`

**Review by ejguan — CHANGES_REQUESTED** [review]
I don't think this is going to work. Can you please do a test?

File: beta (hunk 1 above)
Comment (NivekT) [reply beta]: Updated

**Comment by NivekT** [comment]
> I don't think this is going to work. Can you please do a test?

Updated as symbolic link

**Review by ejguan — APPROVED** [review]

**Pull request merged**